### PR TITLE
Review azs-how-create-load-balancer

### LINKED ISSUE #1637: 28/02/2021 - How to create a load balancer for Azure Stack Hub - Review required
:page_facing_up: Title - How to create a load balancer for Azure Stack Hub
:calendar: Expires - 28/02/2021
:family: Team - Azure Stack Review Team
:package: Product - UKCloud for Microsoft Azure
:link: [GitHub Link](https://github.com/UKCloud/documentation/tree/master/articles/azure/azs-how-create-load-balancer.md)
:link: [Knowledge Centre Link](https://docs.ukcloud.com/articles/azure/azs-how-create-load-balancer.html)

**Before making any changes, make sure you're familiar with our [guidelines](https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html) for contributing to Knowledge Centre articles.**

### PR DESCRIPTION
**Pull request by wturner123**
Closes #1637

### CODE REVIEW

**Review by cblackuk — APPROVED** [review]
LGTM

File: articles/azure/azs-how-create-load-balancer.md [hunk 1]
```diff
@@ -92,23 +90,29 @@ In this section we will create a virtual network and two virtual machines which
...
+        - **IPv4 address space** - The virtual network's address range in CIDR notation (for example, 192.168.1.0/16).
 
-   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0/24). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+        - **Subnet name** - The name of the first subnet within the virtual network.
 
-     ![Create new virtual network](images/azs-browser-create-virtual-network.png)
+        - **Subnet address range** - The subnet's address range in CIDR notation (for example, 192.168.1.0/24). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
 
-5. Click **Create**.
+      ![Create new virtual network - IP Addresses](images/azs-browser-create-virtual-network2.png)
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-load-balancer.md:113]: should the subscription be greyed out like in other screenshots?

Comment (wturner123) [reply articles/azure/azs-how-create-load-balancer.md:113]: Yes it should, good spot. I've updated the image.

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**